manifests/init.pp

```diff
@@ -182,6 +183,7 @@
   String                                               $redis_server_host,
   Integer[1,65536]                                     $redis_server_port,
   Integer[1]                                           $redis_timeout,
+  String                                               $unbound_conf_d,
```

Comment:
This needs a default value

Reply:
Default value is in hiera - see file **data/common.yaml **,